nixos/modules/programs/gnupg.nix

```diff
@@ -90,43 +90,47 @@ in
     };
   };
 
-  config = mkIf cfg.agent.enable {
-    programs.gnupg.agent.pinentryFlavor = mkDefault defaultPinentryFlavor;
+  config = mkMerge [
+    # unconditionally set a default, so it can be used outside this
+    # module (eg by yubikey-agent)
+    { programs.gnupg.agent.pinentryFlavor = mkDefault defaultPinentryFlavor; }
```

Comment:
you can just set a `default = defaultPinentryFlavor` in the options list above (https://github.com/NixOS/nixpkgs/pull/92936/files#diff-46c0e5e66d829e49bf18bc90891c2c90L72), and not use the `mkMerge` and the reformatting.

Reply:
Huh, I just assumed there was some reason it wasn’t done that way originally (to avoid infinite recursion loops?) but if that works I can just do that. I’m fairly new to nixos modules and options and they scare me a bit 😰